### PR TITLE
py-cryptography: update to openssl3 now that dependencies are also up…

### PR DESCRIPTION
**Pull request by jhoyt4**
#### Description

Updates py-cryptography to openssl 3 now that Cairo is updated.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 11.6 20G165 x86_64
Xcode 13.1 13A1030d

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by mascguy** [comment]
FYI, just updated this port to fix an upstream stealth update. So you'll need to rebase against master, and bump your revision to 3.

File: python/py-cryptography/Portfile
Comment (mascguy) [inline python/py-cryptography/Portfile:10]: Since you're only changing `py27-cryptography`, you shouldn't change the overall revision for the port.

Instead, you'll want to bump the revision specific for the Python 2.7 subport, on line 47. And it'll need to be changed to 3, as master is now at 2.

**Comment by mascguy** [comment]
It looks like you just did a merge commit. Instead, you'll want to drop that, rebase against master, and then force-push.

**Review by mascguy — APPROVED** [review]

**Comment by mascguy** [comment]
Already taken care of

**Comment by jhoyt4** [comment]
> Already taken care of

you're fast :)

**Comment by mascguy** [comment]
You're welcome. Let's wait for the CI jobs to complete though, before merging this.

**Comment by mascguy** [comment]
`py27-crytography` is failing to build with OpenSSL 3, just as it did before. Can you confirm that it's building for you locally?

**Review by mascguy — CHANGES_REQUESTED** [review]
`py27-cryptography` fails to build with OpenSSL 3. So unless you can provide an appropriate patch/fix, I'm not sure this is an option...

**Comment by jhoyt4** [comment]
> `py27-cryptography` fails to build with OpenSSL 3. So unless you can provide an appropriate patch/fix, I'm not sure this is an option...

crud - py38/py39 built fine.  I'm going to pull this back.

**Comment by jhoyt4** [comment]
crud - I was having issues with py38/39 pulling in openssl when I opened this.  Will have to go back to the drawing board.

Sorry for the distraction.

**Pull request closed without merging**
